src/app.jsx

```diff
@@ -5,6 +5,14 @@ if (typeof __TEST__ === 'undefined') {
 }
 
 const $ = require('jquery');
+
+const Router = require('react-router'); // or var Router = ReactRouter; in browsers
```

Comment:
Webpack will actually package all of this up and make it browser-safe when it writes to the `dist` directory. :)